shared/components/NextCommunityEvent/index.js

```diff
@@ -42,46 +42,79 @@ export function getHeaderText(startDateTime, endDateTime) {
   }
 }
 
-const NextCommunityEvent = (featuredEvent) => {
-  const { title, startDateTime, endDateTime, location, talks, calendarURL } = featuredEvent;
-  const statusProps = getTicketStatusOptions(featuredEvent);
-  return (
-    <section className="NextCommunityEvent block">
-      <div className="content">
-        <h2 className="NextCommunityEvent__header">
-          {getHeaderText(startDateTime, endDateTime)}
-        </h2>
-        <article className="NextCommunityEvent__section-container">
-          <div className="NextCommunityEvent__section NextCommunityEvent__section__details">
-            <h3 className="NextCommunityEvent__details__heading">{title}</h3>
-            <ul className="NextCommunityEvent__details">
-              <li>
-                <ExternalLink
-                  href={calendarURL}
-                  className="NextCommunityEvent__link--date"
-                >
-                  {eventDateAndTime(startDateTime, endDateTime)}
-                </ExternalLink>
-              </li>
-              <li>
-                <ExternalLink
-                  className="NextCommunityEvent__link--place"
-                  href={googleMapsUrl(location)}
-                  target="_blank"
-                  rel="noopener"
-                >
-                  {eventLocation(location)}
-                </ExternalLink>
-              </li>
-            </ul>
-          </div>
-          <TicketStatus {...statusProps} />
-        </article>
-      </div>
-      <Talks talks={talks} />
-    </section>
-  );
-};
+class NextCommunityEvent extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { collapsed: true };
+  }
+
+  expandInfo(e) {
+    this.setState({ collapsed: false });
+    e.preventDefault();
+  }
+
+  readmoreButton() {
```

Comment:
This one is a good candidate for a separate component. It's ok to have small components.